### PR TITLE
randomize secrets when initializing env

### PR DESCRIPTION
**Pull request by ryanatkn**
Security audit work - it's a footgun to init users with static secrets. This makes 🎲 a placeholder for a random character in the example env files.

Renames the initial env files from `.default` to `.example` to follow the SvelteKit conventions. (gitignore already allows the `*.example` versions)

I think I like the idea of renaming things like `COOKIE_KEYS` to have a `_SECRET` suffix so it's very easy to see sensitive values.

### CODE REVIEW

File: src/lib/server/env.ts [hunk 1]
```diff
@@ -29,17 +36,29 @@ const envs: Array<{file: string; defaultFile: string; load: boolean}> = [
...
+	const contents = example_contents.replaceAll('🎲', () => random_char());
+	await writeFile(env_file.path, contents, 'utf8');
+};
+
+// TODO refactor
+const chars = 'abcdefghijklmnopqrstuvwxyz'
+	.split('')
+	.flatMap((v) => [v, v.toUpperCase()])
+	.concat('0123456789'.split(''));
+const random_char = () => random_item(chars);
```
Comment (ryanatkn) [inline src/lib/server/env.ts:60]: this is somewhat opinionated so I just inlined it here, I didn't see an existing module that made more sense

File: src/lib/infra/.env.production.example [hunk 2]
```diff
@@ -0,0 +1,16 @@
+PUBLIC_GIT_HASH=main
+COOKIE_KEYS=latest_secret_🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲__older_secret_🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲__oldest_secret_🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲🎲
```
Comment (ryanatkn) [inline src/lib/infra/.env.production.example:2]: GitHub didn't pick up the rename, the only thing that changed here is adding the dice emoji

File: src/lib/server/env.ts [hunk 3]
```diff
@@ -29,17 +36,29 @@ const envs: Array<{file: string; defaultFile: string; load: boolean}> = [
...
 };
 
+const init_env_file = async (env_file: Env_File): Promise<void> => {
+	const example_contents = await readFile(env_file.example_path, 'utf8');
+	const contents = example_contents.replaceAll('🎲', () => random_char());
+	await writeFile(env_file.path, contents, 'utf8');
+};
+
+// TODO refactor, the
+const random_char = (chars = alphanumeric_chars) => random_item(chars);
```
Comment (ryanatkn) [inline src/lib/server/env.ts:56]: this is opinionated so I just inlined it here, I didn't see an existing module that made more sense

File: src/lib/docs/setup_dev_environment.svelte [hunk 4]
```diff
@@ -133,11 +133,8 @@ gro # print available tasks`}
 		</li>
 	</ul>
 	<p>
-		Felt's projects are not currently being tested on Windows.
-		<a href="https://docs.microsoft.com/en-us/windows/wsl/wsl2-install">WSL2</a> provides a decent
-		experience running a Linux distro on Windows via the VSCode extension
-		<a href="https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl"
-			>Remote - WSL</a
-		>. (may not work with VSCodium, and may have various issues)
+		Felt chooses Bash integration over Windows support. We recommend <a
```
Comment (greatbacon) [inline src/lib/docs/setup_dev_environment.svelte:136]: it's true lol

**Review by greatbacon — APPROVED** [review]
🎲

**Pull request merged**